doc for insert-update (#291)
arangodb/docs: 3.7/aql/operations-insert.md

  } INTO users OPTIONS { overwrite: true }
```

To update existing documents with documents having the same key there is the
*overwriteMode* query option, which implicitly enables the *overwrite* option .

> This will let you chose between "replace" and "update" overwrite mode. In case

Reviewer: ```suggestion
This will let you choose between "replace" and "update" overwrite mode. In case
```

the *overwriteMode* is "update" the *keepNull* and *mergeObjects* options will

> modify how the replacement is done. Please consult the UPDATE documentation for

Reviewer 2: Not sure if this is the way to go... Maybe rather copy the description to here...

```suggestion
modify how the replacement is done. See the [UPDATE operation](operations-update.html#setting-query-options)
```

> more detail on these options.

Reviewer 2: ```suggestion
for more detail on these options.
```
